content/blog/ebpf-http2-tracing/index.md

```diff
@@ -0,0 +1,145 @@
+---
+path: '/ebpf-http2-tracing'
```

Comment:
We'll need to update the pixie-demo code to use `ebpf-http2-tracing`.
The current code in pixie-demos uses `http2-tracing`.
It's easier to update the code since it was only referenced once.

Reply:
Can you submit a PR for this and I will merge it?

Reply:
https://github.com/pixie-io/pixie-demos/pull/28
Done in this PR.